lib/trino-parquet/src/test/java/io/trino/parquet/writer/TestParquetWriter.java

```diff
@@ -29,9 +29,11 @@ public void testCreatedByIsParsable()
             throws VersionParser.VersionParseException, IOException
     {
         String createdBy = ParquetWriter.formatCreatedBy("test-version");
+        // created-by must start with "parquet-mr" to make CDH 5 perform timezone conversion on INT96 timestamps correctly
+        assertThat(createdBy).startsWith("parquet-mr");
```

Comment:
I propose that

- we keep createdBy as it used to be
- when running test on CDH5, assert it fails (or returns wrong data)
- send follow up PR to remove testing on CDH5

Reply:
I found that the reason only CDH5 failed was that it contains a change from Apache Hive 3.2 https://issues.apache.org/jira/browse/HIVE-21888 which enables `hive.parquet.timestamp.skip.conversion` by default.
When `hive.parquet.timestamp.skip.conversion is disabled, Hive does not look into the createdBy field and always performs conversion.
I added a test and confirmed that HDP3 would also fail if we explicitly enabled `hive.parquet.timestamp.skip.conversion` there (it's disable by default in that version).
So it seems that we should keep the createdBy change to avoid incompatibility with Apache Hive 3.2 and newer as well.